### DISCUSSION TITLE
How is this project different from continue.dev(which is also open source project)?

### DISCUSSION THREAD

**Discussion by Greatz08** [rjmacarthy/twinny#256] [General]
I and many open source lovers have been using continue for months so i recently came across this project which is doing same thing but lack features so one major question arises in my mind that how is it different in many ways compared to that continue project and i guess this is the best place to ask this question so hoping to get detailed response on this

**Comment by rjmacarthy** [comment]
Hello @HakaishinShwet thanks for the interest in this project.

When I started this project some of the similar projects had no/limited for both chat and auto-complete using localhost models. The idea from a personal point of view, at least at the beginning was to implement an easy solution which was totally private and free.  
The solution was to avoid mandatory authentication and login, and by doing so it makes it easier for individuals without access to cloud services.  It's was also clear that this project should not collect data and be fully open source.

When you ask about how this project is different, I simply ask that you try both and decide for yourself.

I hope that answers your questions!

**Reply by Greatz08** [reply]
Yea I also don't like authentication stuff and collection of data and want simplicity with best possible features so I will give it a try.
Thanks for this great project :+))
